test/evergreen.yml

```diff
@@ -956,6 +1016,7 @@ tasks:
             fi
 
   - name: format
+    tags: ["pull_request", "windows"]
```

Comment:
May be better to explicitly use "windows_only" and in the future, we'll have "linux_only" tests such as "s_all"

Reply:
OK, will update the tag name as suggested.